CHANGELOG.md

```diff
@@ -49,6 +49,7 @@
 - `[jest-reporters]` Adds ability to pass options to the istanbul-reporter through `coverageReporters` ([#9572](https://github.com/facebook/jest/pull/9572))
 - `[jest-runtime]` Require stack when a module cannot be resolved ([#9681](https://github.com/facebook/jest/pull/9681))
 - `[jest-transform]` `writeCacheFile` no longer calls `fsync` ([#9695](https://github.com/facebook/jest/pull/9695))
+- `[jest-circus]` Enable writing async test event handlers ([#9392](https://github.com/facebook/jest/pull/9392))
```

Comment:
this should be under master at the top (and sorted alphabetically), but don't worry about it now - I can fix when merging this so you don't have to keep that part updated

Reply:
Sorry, hopefully, I have resolved this now. It looks like I have been resolving merge conflicts with the master branch not carefully here.